src/components/tic-tac-toe.test.tsx

```diff
@@ -0,0 +1,278 @@
+import 'jsdom-global/register'; // at the top of file , even  , before importing react
+import * as React from 'react';
+import { render, mount } from 'enzyme';
+// import { JSDOM } from 'jsdom';
+// const doc = JSDOM.jsdom('<!doctype html><html><body></body></html>'); // Needed for mounting
+
+const TicTacToe = require('./tic-tac-toe');

```

Comment:
Don't mix `import` and `require`; stick with `import` here. Also, it's nicer to `import { Row, Board } from './tic-tac-toe'` rather than having to `<TicTacToe.Blah />`.